backend/de.metas.business/src/main/java/de/metas/product/model/interceptor/M_Product.java

```diff
@@ -66,4 +73,37 @@ private void createOrUpdateProductPlanningsForSelector(final @NonNull I_M_Produc
 		productPlanningSchemaBL.createOrUpdateProductPlanningsForSelector(productId, orgId, productPlanningSchemaSelector);
 	}
 
+	@ModelChange(timings = { ModelValidator.TYPE_BEFORE_NEW, ModelValidator.TYPE_BEFORE_CHANGE },
```

Comment:
```suggestion
	@ModelChange(timings = ModelValidator.TYPE_BEFORE_CHANGE ,
```
if the product is new, there can't be any `C_UOM_Conversion` yet..